src/PhTestHtmlTemplate.php

```diff
@@ -260,14 +258,16 @@ function Html_template($total_suites, $total_cases, $failed_cases, $successful_c
             <!-- End of Content Wrapper -->
       
             <!-- Bootstrap core JavaScript-->
-            <script src="assets/bootstrap-sb-admin-2/js/jquery/jquery.min.js"></script>
-            <script src="assets/bootstrap-sb-admin-2/js/bootstrap.bundle.min.js"></script>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.3/jquery.min.js" integrity="sha512-STof4xm1wgkfm7heWqFJVn58Hm3EtS31XFaagaa8VMReCXAkQnJZ+jEy8PCC/iT18dFy95WcExNHFTqLyp72eQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.2.1/js/bootstrap.bundle.js" integrity="sha512-4WQnCRyZ0CILKrMrO1P40yJrObxaNBOuImuSCFRV47/CWYh3ISyVPmqZnhiZ4OmhHstEj+QaoMDpQo5SnOXDAw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
       
             <!-- Core plugin JavaScript-->
-            <script src="assets/bootstrap-sb-admin-2/js/jquery-easing/jquery.easing.min.js"></script>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.4.1/jquery.easing.min.js" integrity="sha512-0QbL0ph8Tc8g5bLhfVzSqxe9GERORsKhIn1IrpxDAgUsbBGz/V7iSav2zzW325XGd1OMLdL4UiqRJj702IeqnQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>  
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/startbootstrap-sb-admin-2/4.1.4/js/sb-admin-2.js" integrity="sha512-M82XdXhPLLSki+Ld1MsafNzOgHQB3txZr8+SQlGXSwn6veeqtYhPLbQeAfk9Y/Q9/gXcfa1jWT4YYUeoei6Uow==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/startbootstrap-sb-admin-2/4.1.4/js/sb-admin-2.min.js" integrity="sha512-+QnjQxxaOpoJ+AAeNgvVatHiUWEDbvHja9l46BHhmzvP0blLTXC4LsvwDVeNhGgqqGQYBQLFhdKFyjzPX6HGmw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/startbootstrap-sb-admin-2/4.1.4/js/sb-admin-2.min.js" integrity="sha512-+QnjQxxaOpoJ+AAeNgvVatHiUWEDbvHja9l46BHhmzvP0blLTXC4LsvwDVeNhGgqqGQYBQLFhdKFyjzPX6HGmw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
       
             <!-- Custom scripts for all pages-->
-            <script src="assets/bootstrap-sb-admin-2/js/sb-admin-2.min.js"></script>   
             <script src="assets/js/views/test_report_index.js"></script>
```

Comment:
@ruthbri este script sigue siendo externo, cuando el reporte está en otra app, este script no se encuentra.

Fijate en `Do not put the JS code in PHP, read the JS file and put the contents in the right place of the HTML as a <script> tag.`

El render report debe leer el archivo JS y poner su contenido dentro del HTML. O sea, desde PHP cargar el archivo test_report_index.js, leer su contenido como string, y poner ese string en el HTML dentro de `<script>...</script>`, sino no funciona.

Y la idea de leer el archivo es para evitar poner código JS dentro del PHP, hablamos de eso hace un par de semanas, de evitar poner JS o HTML dentro de funciones PHP porque es difícil de mantener.

Con eso ya estaría para mergear.